DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -71,6 +71,7 @@ namespace pat {
   typedef std::vector<edm::FwdPtr<reco::BaseTagInfo> > TagInfoFwdPtrCollection;
   typedef std::vector<edm::FwdPtr<reco::PFCandidate> > PFCandidateFwdPtrCollection;
   typedef std::vector<edm::FwdPtr<CaloTower> > CaloTowerFwdPtrCollection;
+  typedef std::vector<edm::Ptr<pat::Jet> > JetPtrCollection;
```

Comment:
Maybe `edm::PtrVector<pat::Jet>` instead of `std::vector<edm::Ptr<pat::Jet> >` since subjets of particular type will be from the same collection? Pn the other hand, typically there are only a few subjets per fat jet so one would not save that much space which this type of change.